src/item_factory.cpp

```diff
@@ -49,6 +49,8 @@ static t_string_set item_blacklist;
 static t_string_set item_whitelist;
 static bool item_whitelist_is_exclusive = false;
 
+static DynamicDataLoader::deferred_json deferred;
```

Comment:
This causes initialization of the json system before anything else is loaded. Even if it isn't a problem by itself, it could cause some of the problems with debugging.